---
abstract: While DNABERT leverages k-mer embeddings to model genomic sequences, its
  exclusive reliance on nucleotide k-mers can limit its effectiveness in capturing
  regulatory elements that lack distinct motif signals or display subtle, compositionally
  diffuse patterns. In this study, we explore a multimodal approach by augmenting
  DNABERT embeddings with DNA-intrinsic featuresâ€”including nucleotide composition,
  purine-pyrimidine balance, CpG density, and structural properties such as minor
  groove width and electrostatic potential. These physicochemical and sequence-derived
  features offer complementary information about DNA shape and stability, often critical
  in regulatory regions such as non-TATA promoters and certain transcription factor
  binding sites (TFBS). By integrating these features with DNABERT representations,
  we show improved model performance in terms of overall prediction accuracy and ability
  to interpret pattern-depleted regulatory sequences. We applied this framework, DNABERT-CoreProm-MM
  (Core Promoter model with MultiModalities), to the task of promoter prediction,
  with a focus on both TATA and non-TATA Core promoter sequences. Our results demonstrate
  that the DNABERT-CoreProm-MM model improves prediction accuracy by 3.72% for TATA
  promoters and 22.56% for non-TATA promoters. These findings highlight the value
  of sequence-intrinsic and shape feature multimodalities in enhancing the interpretability
  and accuracy of transformer-based models, particularly for genomic sequences lacking
  strong motif structure. This approach offers a more comprehensive and biologically
  informed framework for modeling DNA regulatory elements.
title: Augmenting DNABERT Embeddings with Multimodal DNA Features for Improved Regulatory
  Sequence Interpretation
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: papineni25a
month: 0
tex_title: Augmenting DNABERT Embeddings with Multimodal DNA Features for Improved
  Regulatory Sequence Interpretation
firstpage: 294
lastpage: 303
page: 294-303
order: 294
cycles: false
bibtex_author: Papineni, Nimisha and Dutta, Pratik and Chao, Max and Acanto, Orbin
  and Sathian, Rekha and Surana, Pallavi and Davuluri, Ramana
author:
- given: Nimisha
  family: Papineni
- given: Pratik
  family: Dutta
- given: Max
  family: Chao
- given: Orbin
  family: Acanto
- given: Rekha
  family: Sathian
- given: Pallavi
  family: Surana
- given: Ramana
  family: Davuluri
date: 2025-12-31
address:
container-title: Proceedings of the 20th Machine Learning in Computational Biology
  meeting
volume: '311'
genre: inproceedings
issued:
  date-parts:
  - 2025
  - 12
  - 31
pdf: https://raw.githubusercontent.com/mlresearch/v311/main/assets/papineni25a/papineni25a.pdf
extras:
- label: Supplementary PDF
  link: https://raw.githubusercontent.com/mlresearch/v311/main/assets/assets/papineni25a/papineni25a-supp.pdf
# Format based on Martin Fenner's citeproc: https://blog.front-matter.io/posts/citeproc-yaml-for-bibliographies/
---
