---
abstract: Deep neural networks have been transforming the field of bioinformatics
  and computational biology in recent years, especially in genomic-related tasks.
  Trained neural networks have enabled unprecedented capabilities in predicting molecular
  and genomic phenotypes. A fundamental step following deep-neural-network training
  and performance evaluation is the interpretation of the trained neural networks
  to learn new biology and validate the trained models. Techniques like integrated
  gradients have been highly successful in local interpretability, i.e., attributing
  importance to a specific residue in a given DNA, RNA, or amino acid sequence. But,
  there still remains the challenge of finding the global patterns that are shared
  among many sequences to understand the biological mechanism. Currently, TF-MoDISco
  is the only available method for this task. However, TF-MoDISco takes hours to run
  on standard datasets, and it reports many redundant and false motifs. Here, we present
  F-MoDA (Fourier-based Motif Discovery in Attribution maps), a novel computational
  method for efficiently and accurately discovering shared sequence motifs in residue-level
  attribution maps. F-MoDA leverages signal processing techniques and a hierarchical
  clustering approach to identify recurring regulatory patterns. We evaluated F-MoDA
  against TF-MoDISco over an established motif-finding benchmark and found that F-MoDA
  reports motifs that are more similar to the ground truth, in addition to reporting
  fewer redundant motifs and fewer false motifs. Moreover, F-MoDA runs much faster
  and uses less memory. We expect F-MoDA to be utilized in many studies applying deep
  neural networks to genomics data. F-MoDA is publicly available at https://github.com/OrensteinLab/F-MoDA.
title: 'F-MoDA: Efficient Fourier-based motif discovery in attribution maps'
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: yaish25a
month: 0
tex_title: 'F-MoDA: Efficient Fourier-based motif discovery in attribution maps'
firstpage: 326
lastpage: 335
page: 326-335
order: 326
cycles: false
bibtex_author: Yaish, Ofir and Orenstein, Yaron
author:
- given: Ofir
  family: Yaish
- given: Yaron
  family: Orenstein
date: 2025-12-31
address:
container-title: Proceedings of the 20th Machine Learning in Computational Biology
  meeting
volume: '311'
genre: inproceedings
issued:
  date-parts:
  - 2025
  - 12
  - 31
pdf: https://raw.githubusercontent.com/mlresearch/v311/main/assets/yaish25a/yaish25a.pdf
extras:
- label: Supplementary PDF
  link: https://raw.githubusercontent.com/mlresearch/v311/main/assets/assets/yaish25a/yaish25a-supp.pdf
# Format based on Martin Fenner's citeproc: https://blog.front-matter.io/posts/citeproc-yaml-for-bibliographies/
---
