---
abstract: Single-cell Hi-C (scHi-C) enables the study of 3D genome organization at
  the resolution of individual cells and cell types that cannot be isolated for bulk
  profiling. However, the extreme sparsity of scHi-C data presents major challenges,
  particularly in recovering cell-type-specific 3D structures when only a small number
  of cells are available. We introduce contactVI, a method that combines the strengths
  of graph-based models and variational autoencoders (VAEs) to account for spatial
  dependencies in noisy chromatin interaction data and effectively denoise them. On
  simulated data, contactVI outperforms existing imputation methods in recovering
  Hi-C contact maps at both the single-cell and cell-type levels. On real datasets,
  contactVI performs comparably to or better than other graph-based methods across
  different resolutions. When applied to jointly profiled single-cell Hi-C and RNA-seq
  data, contactVI successfully recovers the expected association between genome compartmentalization
  and gene expression.
title: Variational Graph Auto-encoder for Denoising Single-cell Hi-C Data
layout: inproceedings
series: Proceedings of Machine Learning Research
publisher: PMLR
issn: 2640-3498
id: shokraneh-kenari25a
month: 0
tex_title: Variational Graph Auto-encoder for Denoising Single-cell Hi-C Data
firstpage: 213
lastpage: 229
page: 213-229
order: 213
cycles: false
bibtex_author: Shokraneh Kenari, Neda and Libbrecht, Maxwell
author:
- given: Neda
  family: Shokraneh Kenari
- given: Maxwell
  family: Libbrecht
date: 2025-12-31
address:
container-title: Proceedings of the 20th Machine Learning in Computational Biology
  meeting
volume: '311'
genre: inproceedings
issued:
  date-parts:
  - 2025
  - 12
  - 31
pdf: https://raw.githubusercontent.com/mlresearch/v311/main/assets/shokraneh-kenari25a/shokraneh-kenari25a.pdf
extras: []
# Format based on Martin Fenner's citeproc: https://blog.front-matter.io/posts/citeproc-yaml-for-bibliographies/
---
